### ISSUE TITLE
Mozilla's Add-on Repository 

### ISSUE THREAD

**Issue by dev-a** [rharel/webext-private-bookmarks#187]
Apparently the firefox store is not at all synchronized with the github.
Indeed, when we install on firefox we have the version 0.1.19a when the github is at version 0.2.20
and firefox says "last update" 3 years ago (Jul 13, 2019)

**Comment by WolfganP** [comment]
I noticed the same version mismatch as well. Is there any xpi file somewhere to manually update to latest dev build?

**Comment by rharel** [comment]
That's true, I haven't updated the published version for a while because it's mostly working fine aside from the occasional mystery bug you see some people have posted about here. That one seems to coincide with something going wrong during the interaction of a Firefox update and the addon, but if you backup your bookmarks often that should keep you covered. 

The latest version of the addon I work on in little windows of time that I find for it, but as you can see that's not often these days. Life has grown too busy for me to have much spare time to work on this project. There is no development xpi available, but I can upload one next weekend if you really want one, that doesn't cost much time. Though you should know it being a dev build means it's not been tested as thoroughly.

**Comment by rharel** [comment]
Good idea, I should read about GitHub actions and see how to automate the build.

**Comment by WolfganP** [comment]
Found a workflow to build the crx and xpi at https://github.com/cardinalby/memrise-audio-uploader/blob/master/.github/workflows/build-assets-on-release.yml with a good explanation of how it was build at https://dev.to/cardinalby/webextension-deployment-and-publishing-using-github-actions-522o
I hope it helps!
